editor/store-defaults.js

```diff
@@ -0,0 +1,7 @@
+export const STORE_DEFAULTS = {
```

Comment:
not sure if this should be in a separate file, what about just a module constant in `store.js`

Reply:
That's tricky, because `store.js` imports from `store-persist.js`, and if we moved it into `store.js` we end up with circular imports. So because we have the reducer importing it as well, a separate file seemed to be the cleanest way.

Reply:
I guess this would create a cyclic dependency between `store`  and `reducer` so maybe in `reducer` instead

Reply:
Seems weird to have _store_ defaults in reducer to me